students/chrissp/lesson02/series.py

```diff
@@ -0,0 +1,65 @@
+def fibonacci(n):
+    """ This function will compute the nth Fibonacci number. """
+    return sum_series(n, 0, 1)
+
+
+def lucas(n):
+    """ This function will compute the nth Lucas number. """
+    return sum_series(n, 2, 1)
+
+
+def sum_series(n, n0 = 0, n1 = 1):
```

Comment:
note that there should be no spaces aroun `=` operator
`def sum_series(n, n0=0, n1=1)`